cmd/frontend/graphqlbackend/highlight.go

```diff
@@ -31,3 +39,87 @@ func fromVCSHighlights(vcsHighlights []git.Highlight) []*highlightedRange {
 	}
 	return highlights
 }
+
+func ParseLinesFromHighlight(input string) (map[int32]string, error) {
+	doc, err := html.Parse(strings.NewReader(input))
+	if err != nil {
+		return nil, err
+	}
+
+	lines := make(map[int32]string)
+
+	body := doc.FirstChild.LastChild // html->body
+	pre := body.FirstChild
+	if pre == nil || pre.Type != html.ElementNode || pre.DataAtom != atom.Pre {
+		return nil, fmt.Errorf("expected html->body->pre, found %+v", pre)
+	}
+
+	var next = pre.FirstChild
+	var line int32 = 1
+	var codeCell *html.Node = &html.Node{Type: html.ElementNode, DataAtom: atom.Div, Data: atom.Div.String()}
```

Comment:
```suggestion
	var (
		next                = pre.FirstChild
		line int32          = 1
		codeCell *html.Node = &html.Node{Type: html.ElementNode, DataAtom: atom.Div, Data: atom.Div.String()}
	)
```

Kinda like this. You need to run `go fmt` on it though :)